### PR TITLE
Try harder not to log sensitive information

### PR DESCRIPTION
**Pull request by a1exsh**
Internal ticket: 898

### CODE REVIEW

**Comment by a1exsh** [comment]
:+1:

File: server/logger.js [hunk 1]
```diff
@@ -47,9 +47,9 @@ const winstonInstance = new winston.Logger({
  */
 const loggerConf = {
     winstonInstance: winstonInstance,
-    dynamicMeta: getMeta,
```
Comment (antban) [inline server/logger.js:50]: Why you removed dynamic meta?
How will it be possible to see who is executing requests in the logs?

Comment (antban) [reply server/logger.js:50]: @a1exsh

Comment (a1exsh) [reply server/logger.js:50]: I was outraged that it also mucks with the access token.  I guess I've missed the part where it adds the user.  I will put it back, but omit the token hash.

**Comment by antban** [comment]
:+1:

**Comment by a1exsh** [comment]
:+1:

**Comment by a1exsh** [comment]
:+1:

**Comment by antban** [comment]
:+1:

**Pull request merged**